browser/ui/android/strings/android_brave_strings.grd

```diff
@@ -2398,7 +2398,10 @@ Are you sure you want to do this?
       Protect 5 devices on 1 subscription
     </message>
     <message name="IDS_TRIAL_TEXT" desc="Brave VPN plan text">
-      All plans include a free 7-day trial!
+      All plans include a <ph name="TRIAL_DAYS">%1$s</ph>
+    </message>
+    <message name="IDS_DAY_7_TRIAL_TEXT" desc="Brave VPN plan text">
+      free 7-day trial!
```

Comment:
Is there no way of doing the underlining without splitting the text into 2 strings? Something along the lines of https://source.chromium.org/chromium/chromium/src/+/main:chrome/browser/search_engines/android/java/src/org/chromium/chrome/browser/search_engines/SogouPromoDialog.java;l=109?

Reply:
@mkarolin we need to apply underline for partial text. so we need to get the start and end of the character.

Reply:
Bummer, it's not great for l10n to split strings like that.